src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,29 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+
+    private int bladeLength;
```

Comment:
Don't forget about access modifiers for your custom fields. It's better to make `bladeLength` and `bladeWidth` private and access them through getters and setters.